frontend/packages/dev-console/src/components/pipelineruns/PipelineRunResourceList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { DevPreviewBadge } from '@console/shared';
+import {
+  pipelineRunFilterReducer,
+  pipelineRunStatusFilter,
+} from '../../utils/pipeline-filter-reducer';
+import { PipelineRunModel } from '../../models';
+import PipelineRunsList from './PipelineRunList';
+
+const filters = [
+  {
+    type: 'pipelinerun-status',
+    selected: ['Succeeded'],
```

Comment:
I am not sure about just selecting `Succeeded`... I've reached out to UX. Personally I don't like it, so I want to make sure it's intentional and desired we mimic the logic from the pipeline -> pipeline-run page.

Reply:
cc @serenamarie125

Reply:
@abhinandan13jan I've got confirmation that we want all of them selected. Could you make this change to this page as well as the Pipeline's PipelineRun Page? Ideally, making use of the same data-object between the two.

cc @serenamarie125

Reply:
Updated this